test/conf/protractor.local.desktop.conf.js

```diff
@@ -28,27 +12,20 @@ config.multiCapabilities = [
 		chromeOptions: {
 			args: [ 'disable-infobars' ]
 		},
-	},
-	// {
-	// 	browserName: 'chrome',
-	// 	logName: 'chrome-headless-latest',
-	// 	shardTestFiles: true,
-	// 	chromeOptions: {
-	// 		args: [ '--headless' ]
-	// 	},
-	// },
+	}
 ];
 
 config.plugins = [
 	{
-		path: join(process.cwd(), './build/index.js'),
+		path: resolve(process.cwd(), './build/index.js'),
 		options: {
-			baselineFolder: join(process.cwd(), './localBaseline/'),
-			debug: true,
-			formatImageName: `{tag}-{logName}-{width}x{height}`,
-			screenshotPath: join(process.cwd(), '.tmp/'),
+			baselineFolder: resolve(process.cwd(), './test/images/local/baseline'),
```

Comment:
This also need to be set to `localBaseline` because this is for a local execution that doesn't need to be submitted to the repo